### DISCUSSION TITLE
How to fix enter button not working on mobile?

### DISCUSSION THREAD

**Discussion by Play-Tiger** [searxng/searxng#1472] [Q&A]
Is there an option that when you press enter on mobile that you start searching and don't need to press the search button. Because this works fine on PC and even on mobile if you are on the default search page, but if you are on the page with search results (example in screenshot) this doesn't work anymore. Is there a workaround for this issue?
![image](https://user-images.githubusercontent.com/76755673/177771053-7d352207-5d18-422d-b7cf-102908acbb4a.png)

**Comment by CountCypher** [comment]
This seems to be an open issue currently: #1613
